src/patterns/components/dictionary/base.hbs

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
I think we should standardize what we call the default pattern in a collection. I think I usually am more verbose and I have been using "basic". So for this one it would be "Basic Dictionary" and the next "Striped Dictionary", etc.

Reply:
Did we remove the heading from this component?